docarray/document/mixins/mesh.py

```diff
@@ -112,3 +112,32 @@ def load_vertices_and_faces_to_point_cloud(self: 'T', samples: int) -> 'T':
             )
 
         return self
+
+    def load_uris_to_rgbd_tensor(self: 'T') -> 'T':
+        """Load RGB image from :attr:`.uri` of :attr:`.chunks[0]` and the depth image from :attr:`.uri` of :attr:`.chunks[1]` and merge them into :attr:`.tensor`.
+
+        :return: itself after processed
+        """
+        from PIL import Image
+
+        if len(self.chunks) != 2:
+            raise ValueError(
+                f'The provided Document does not have 2 chunks but instead {len(self.chunks)}. To load uris to RGBD tensor, the Document needs to have two chunks, with the first one providing the RGB image uri, and the second one providing the depth image uri.'
+            )
```

Comment:
@JoanFM @alaeddine-13 @JohannesMessner I believe we don't have a choice to do this here but I would like to have your opinion. Main issue is that we need to have two uri